src/local/LocalTransport.ts

```diff
@@ -0,0 +1,93 @@
+import assert from 'assert';
+import _, { values, valuesIn } from 'lodash';
+import debug from 'debug';
+import Matcher from './matcher';
+import AutoBus from '../AutoBus';
+
+const log = debug('autobus:transport:local');
+
+export interface Listener<T> {
+  (message: T): void;
+}
+
+export default class LocalTransport {
+  _bus: AutoBus;
+  _wildcardMatchers: Matcher;
+  _channelListeners: Record<string, Listener<any>[]>;
+
+  constructor(bus: AutoBus) {
+    this._bus = bus;
+    this._channelListeners = {};
+    this._wildcardMatchers = new Matcher();
+  }
+
+  send<T>(channel: string, data: T): void {
+    const self = this;
+    const listeners = this._getListeners(channel);
+    if (listeners.length > 1) {
+      log('More than one listener found');
+      log(data);
+      throw new Error('Can only have one matching listener when sending a message, try publish?');
+    }
+    if (listeners.length == 0) {
+      log('No listeners found for message');
+      log(data);
+    }
+    _.forEach(listeners, function (listener) {
```

Comment:
If we don't need the Lodash functions, I think we should replace them with vanilla Array functions.